service/src/main/java/org/ohdsi/gisservice/config/EncryptionConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.ohdsi.gisservice.config;
+
+import org.apache.commons.lang3.StringUtils;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.jasypt.encryption.StringEncryptor;
+import org.jasypt.encryption.pbe.PBEStringEncryptor;
+import org.jasypt.encryption.pbe.StandardPBEStringEncryptor;
+import org.ohdsi.gisservice.utils.NotEncrypted;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+
+@Configuration
+public class EncryptionConfig {
+
+	private static final Logger logger = LoggerFactory.getLogger(EncryptionConfig.class);
+
+	@Value("${jasypt.encryptor.enabled:false}")
+	private boolean encryptorEnabled;
+
+	@Bean
+	public StringEncryptor defaultStringEncryptor(Environment env) {
```

Comment:
Shouldn't we create a single config in commons and reuse across projects? I believe it is duplicated now

Reply:
Sure, I guess the best candidate to move to is arachne-commons:datasource-manager.